src/garage/tf/embeddings/gaussian_mlp_encoder.py

```diff
@@ -127,8 +127,10 @@ def build(self, embedding_input, name=None):
             self._variable_scope = vs
             self._dist, _, _ = self.model.build(embedding_input, name=name)
             self._f_dist = tf.compat.v1.get_default_session().make_callable(
-                [self._dist.sample(), self._dist.loc,
-                 self._dist.stddev()],
+                [
+                    self._dist.sample(), self._dist.loc,
+                    tf.math.log(self._dist.stddev())
```

Comment:
can't you capture the `mean` `log_std` in the call to build above, rather than referring to `dist.loc` and running `tf.log` every time?

Reply:
That’s right, will fix